v2/nop_jitter.go

```diff
@@ -0,0 +1,11 @@
+package backoff
```

Comment:
Please put this in the same `jitter.go` fiile

Reply:
Do you prefer to move also `random_jitter`?

Reply:
I probably just didn't notice. yeah, just put it in jitter.go, please.